layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_%.bbappend

```diff
@@ -187,18 +193,21 @@ BALENA_CONFIGS_append_jetson-nano = " gasket"
 BALENA_CONFIGS_append_jetson-nano-emmc = " gasket"
 
 BALENA_CONFIGS_append_photon-nano = " tlc591xx"
+BALENA_CONFIGS_append_photon-tx2-nx = " tlc591xx"
```

Comment:
I assume this is is necessary for the photon tx2 nx too, correct?

Reply:
Yes